instrumentation/logback-classic-1.2/src/main/java/com/nr/agent/instrumentation/logbackclassic12/Logger_Instrumentation.java

```diff
@@ -0,0 +1,18 @@
+package com.nr.agent.instrumentation.logbackclassic12;
+
+import ch.qos.logback.classic.Level;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import org.slf4j.Marker;
+
+@Weave(originalName = "ch.qos.logback.classic.Logger", type = MatchType.Interface)
+public abstract class Logger_Instrumentation {
+    private void buildLoggingEventAndAppend(final String localFQCN, final Marker marker, final Level level, final String msg, final Object[] params,
+            final Throwable t) {
+        Weaver.callOriginal();
+        NewRelic.incrementCounter(Constants.LOG_LINE.getVal());
+        NewRelic.incrementCounter(String.format(Constants.LOG_LINE_LEVEL.getVal(), level));
```

Comment:
If we're concerned about overhead, is there a strong reason to separate the constants out so we have to use an additional method call to get them? I don't see that done elsewhere in the instrumentation commonly.

Reply:
Good point, I'm going to revert this.